chef_master/source/platforms.rst

```diff
@@ -332,9 +256,6 @@ The following table lists the Foundational platforms for the Chef push jobs:
    * -
      - ``x86_64``
      - ``5``, ``6``, ``7``
-   * - Debian
-     - ``i386``, ``x86_64``
-     - ``7``
    * - Red Hat Enterprise Linux
      - ``i386``
```

Comment:
push jobs client also builds/tests on x86_64 for all platforms, fyi. (and OSX x86_64)